webclient/app/src/app/commons/CockpitAppBar.jsx

```diff
@@ -21,10 +31,17 @@ function CockpitAppBar() {
                             <HomeIcon />
                         </IconButton>
                         <Typography variant="h1" component="div" sx={{flexGrow: 1}}>KI Cockpit</Typography>
-                        <>
-                            <ConfigMenu />
-                            <InfoMenu />
-                        </>
+                        <Tooltip title="Incident Map">
+                            <IconButton
+                                onClick={() => {/*TODO*/}}
+
+                                href="./#/incident-map-view"
```

Comment:
* corrected -> needs a # to route to the correct site
* for next time: try to avoid additional styling informations
* avoid empty <></> if not needed